pkg/api/dashboard.go

```diff
@@ -272,33 +272,39 @@ func (hs *HTTPServer) getDashboardHelper(ctx context.Context, orgID int64, id in
 	return queryResult, nil
 }
 
+func (hs *HTTPServer) GetSoftDeletedDashboard(ctx context.Context, orgID int64, uid string) (*dashboards.Dashboard, response.Response) {
```

Comment:
Is this method in the API necessary? You can call the `GetSoftDeletedDashboard` from the service directly or if you want to add this method here it should be private.

Reply:
In that case you can change the first letter to lowercase since it is private and will be used only in this package.

Reply:
I decided to simplify the code and remove that function